packages/ckeditor5-engine/src/model/treewalker.ts

```diff
@@ -183,6 +183,28 @@ export default class TreeWalker implements Iterable<TreeWalkerValue> {
 		}
 	}
 
+	/**
+	 * Moves treewalker {@link #position} to provided `position`. Treewalker will
```

Comment:
```suggestion
	 * Moves tree walker {@link #position} to provided `position`. Tree walker will
```

I know I proposed this but I made my mind and now I think it should be with a space, since classname is `TreeWalker` not `Treewalker`.